### DISCUSSION TITLE
Special link for users

### DISCUSSION THREAD

**Discussion by fayasnoushad** [npm/feedback#600] [Website] [answered]
Please provide special link for users.

Like `https://user.npmjs.com/<username>` redirect to `https://www.npmjs.com/~<username>`

**Comment by MylesBorins** [comment, marked as answer]
Hey @FayasNoushad. Thanks for the suggestion but at this time this is not something that we will be pursuing. /~:username has been used for long enough that we can never change it and I don't think we can justify introducing an additional source of truth at this time

**Reply by fayasnoushad** [reply]
Ok, but a redirect domain is better for that.
